### ISSUE TITLE
`bundle exec jekyll serve` not `npm run dev` is needed, `npm run rel` doesn't work at all `bundle exec jekyll build` is needed

### ISSUE THREAD

**Issue by justingolden21** [kangabru/jekyll-tailwindcss-boilerplate#4]
Also, even after getting it working, this is prod:
![image](https://user-images.githubusercontent.com/30274440/100395952-f17f2000-2ff7-11eb-90c9-e5119d554d87.png)

and this is dev:
![image](https://user-images.githubusercontent.com/30274440/100395960-fa6ff180-2ff7-11eb-918e-63eec2979676.png)

My cmd output:
```cmd
Microsoft Windows [Version 10.0.19042.630]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>rpm run dev
'rpm' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run dev

> your.repo.name@0.0.1 dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:dev && jekyll serve


> your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> postcss assets/tailwind.config.css -o assets/tailwind.css

{ Error: Cannot find module 'cssnano'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\postcss.config.js:4:17)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 css:dev: `postcss assets/tailwind.config.css -o assets/tailwind.css`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 css:dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_51_33_105Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 dev: `npm run css:dev && jekyll serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_51_33_145Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run setup

> your.repo.name@0.0.1 setup C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> gem install bundler && bundle install && npm i && npm i postcss-cli -g

Successfully installed bundler-2.1.4
Parsing documentation for bundler-2.1.4
Done installing documentation for bundler after 3 seconds
1 gem installed
Using public_suffix 4.0.3
Using addressable 2.7.0
Using bundler 2.1.4
Using colorator 1.1.0
Using concurrent-ruby 1.1.6
Using eventmachine 1.2.7 (x64-mingw32)
Using http_parser.rb 0.6.0
Using em-websocket 0.5.1
Using ffi 1.12.2 (x64-mingw32)
Using forwardable-extended 2.6.0
Using i18n 1.8.2
Using sassc 2.2.1 (x64-mingw32)
Using jekyll-sass-converter 2.1.0
Using rb-fsevent 0.10.3
Using rb-inotify 0.10.1
Using listen 3.2.1
Using jekyll-watch 2.2.1
Using kramdown 2.1.0
Using kramdown-parser-gfm 1.1.0
Using liquid 4.0.3
Using mercenary 0.3.6
Using pathutil 0.16.2
Using rouge 3.17.0
Using safe_yaml 1.0.5
Using unicode-display_width 1.7.0
Using terminal-table 1.8.0
Using jekyll 4.0.0
Using jekyll-seo-tag 2.6.1
Bundle complete! 2 Gemfile dependencies, 28 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
npm WARN your.repo.name@0.0.1 No repository field.
npm WARN your.repo.name@0.0.1 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 289 packages from 226 contributors and audited 291 packages in 7.234s

22 packages are looking for funding
  run `npm fund` for details

found 2 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
C:\Users\justi\AppData\Roaming\npm\postcss -> C:\Users\justi\AppData\Roaming\npm\node_modules\postcss-cli\bin\postcss
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules\postcss-cli\node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN postcss-cli@8.3.0 requires a peer of postcss@^8.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN postcss-reporter@7.0.1 requires a peer of postcss@^8.1.0 but none is installed. You must install peer dependencies yourself.

+ postcss-cli@8.3.0
updated 1 package in 3.99s

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run dev

> your.repo.name@0.0.1 dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:dev && jekyll serve


> your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> postcss assets/tailwind.config.css -o assets/tailwind.css

Browserslist: caniuse-lite is outdated. Please run the following command: `npm update`
Traceback (most recent call last):
        10: from C:/Ruby26-x64/bin/jekyll:23:in `<main>'
         9: from C:/Ruby26-x64/bin/jekyll:23:in `load'
         8: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/exe/jekyll:11:in `<top (required)>'
         7: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
         6: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
         5: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `setup'
         4: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `map'
         3: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         2: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         1: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:31:in `block in setup'
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated public_suffix 4.0.6, but your Gemfile requires public_suffix 4.0.3. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 dev: `npm run css:dev && jekyll serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_52_22_195Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>bundle exec npm dev

Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    clean-install, clean-install-test, completion, config,
    create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
    edit, explore, fund, get, help, help-search, hook, i, init,
    install, install-ci-test, install-test, it, link, list, ln,
    login, logout, ls, org, outdated, owner, pack, ping, prefix,
    profile, prune, publish, rb, rebuild, repo, restart, root,
    run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h  quick help on <command>
npm -l            display full usage info
npm help <term>   search for help on <term>
npm help npm      involved overview

Specify configs in the ini-formatted file:
    C:\Users\justi\.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@6.14.9 C:\Users\justi\AppData\Roaming\npm\node_modules\npm


C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm update
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN your.repo.name@0.0.1 No repository field.
npm WARN your.repo.name@0.0.1 No license field.

+ tailwindcss@1.9.6
+ postcss-cli@7.1.2
+ @fullhuman/postcss-purgecss@2.3.0
+ postcss@7.0.35
added 28 packages from 12 contributors, removed 7 packages, updated 42 packages, moved 2 packages and audited 313 packages in 8.278s

29 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run dev

> your.repo.name@0.0.1 dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:dev && jekyll serve


> your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> postcss assets/tailwind.config.css -o assets/tailwind.css

Traceback (most recent call last):
        10: from C:/Ruby26-x64/bin/jekyll:23:in `<main>'
         9: from C:/Ruby26-x64/bin/jekyll:23:in `load'
         8: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/exe/jekyll:11:in `<top (required)>'
         7: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
         6: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
         5: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `setup'
         4: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `map'
         3: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         2: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         1: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:31:in `block in setup'
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated public_suffix 4.0.6, but your Gemfile requires public_suffix 4.0.3. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 dev: `npm run css:dev && jekyll serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_53_31_027Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>bundle exec jekyll serve
Configuration file: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_config.yml
            Source: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate
       Destination: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 0.203 seconds.
  Please add the following to your Gemfile to avoid polling for changes:
    gem 'wdm', '>= 0.1.0' if Gem.win_platform?
 Auto-regeneration: enabled for 'C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate'
    Server address: http://127.0.0.1:4000
  Server running... press ctrl-c to stop.
[2020-11-26 14:54:37] ERROR `/assets/favicon.ico' not found.
Terminate batch job (Y/N)?
^CTerminate batch job (Y/N)?
^C
C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run dev

> your.repo.name@0.0.1 dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:dev && jekyll serve


> your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> postcss assets/tailwind.config.css -o assets/tailwind.css

Traceback (most recent call last):
        10: from C:/Ruby26-x64/bin/jekyll:23:in `<main>'
         9: from C:/Ruby26-x64/bin/jekyll:23:in `load'
         8: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/exe/jekyll:11:in `<top (required)>'
         7: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
         6: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
         5: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `setup'
         4: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `map'
         3: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         2: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         1: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:31:in `block in setup'
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated public_suffix 4.0.6, but your Gemfile requires public_suffix 4.0.3. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 dev: `npm run css:dev && jekyll serve`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_56_22_129Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run rel

> your.repo.name@0.0.1 rel C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:rel && jekyll build


> your.repo.name@0.0.1 css:rel C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:dev -- --env production


> your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> postcss assets/tailwind.config.css -o assets/tailwind.css "--env" "production"


warn - Tailwind is not purging unused styles because no template paths have been provided.
warn - If you have manually configured PurgeCSS outside of Tailwind or are deliberately not removing unused styles, set `purge: false` in your Tailwind config file to silence this warning.
warn - https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
Browserslist: caniuse-lite is outdated. Please run the following command: `npm update`
Traceback (most recent call last):
        10: from C:/Ruby26-x64/bin/jekyll:23:in `<main>'
         9: from C:/Ruby26-x64/bin/jekyll:23:in `load'
         8: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/exe/jekyll:11:in `<top (required)>'
         7: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
         6: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
         5: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `setup'
         4: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `map'
         3: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         2: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         1: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:31:in `block in setup'
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated public_suffix 4.0.6, but your Gemfile requires public_suffix 4.0.3. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 rel: `npm run css:rel && jekyll build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 rel script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_56_42_694Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm update

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm update

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run rel

> your.repo.name@0.0.1 rel C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:rel && jekyll build


> your.repo.name@0.0.1 css:rel C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npm run css:dev -- --env production


> your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> postcss assets/tailwind.config.css -o assets/tailwind.css "--env" "production"


warn - Tailwind is not purging unused styles because no template paths have been provided.
warn - If you have manually configured PurgeCSS outside of Tailwind or are deliberately not removing unused styles, set `purge: false` in your Tailwind config file to silence this warning.
warn - https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
Browserslist: caniuse-lite is outdated. Please run the following command: `npm update`
Traceback (most recent call last):
        10: from C:/Ruby26-x64/bin/jekyll:23:in `<main>'
         9: from C:/Ruby26-x64/bin/jekyll:23:in `load'
         8: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/exe/jekyll:11:in `<top (required)>'
         7: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/jekyll-4.1.1/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
         6: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler.rb:149:in `setup'
         5: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `setup'
         4: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:26:in `map'
         3: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         2: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/spec_set.rb:147:in `each'
         1: from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:31:in `block in setup'
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/bundler-2.1.4/lib/bundler/runtime.rb:312:in `check_for_activated_spec!': You have already activated public_suffix 4.0.6, but your Gemfile requires public_suffix 4.0.3. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 rel: `npm run css:rel && jekyll build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 rel script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-26T22_57_34_705Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>bundle exec jekyll build
Configuration file: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_config.yml
            Source: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate
       Destination: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 0.158 seconds.
 Auto-regeneration: disabled. Use --watch to enable.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>
```

**Comment by justingolden21** [comment]
The problem of the mislinked assets is probably from `_includes/head.html`:

```html
<link rel="stylesheet" href="{{ "/assets/index.css" | relative_url }}">
<link rel="stylesheet" href="{{ "/assets/tailwind.css" | relative_url }}">
<link rel="shortcut icon" type="image/png" href={{ "/assets/favicon.ico" | relative_url }}>
```

Have no idea what's going on with your quotes here, but I believe they should say `assets/index.css` not `/assets/index.css` (delete leading slash)

Still, I have absolutely no idea why `npm run dev` and `npm run rel` don't work, any help greatly appreciated.

**Comment by kangabru** [comment]
Hey Justin, sorry about the issue you're having! Unfortunately I'm not sure why you can't run Jekyll commands (I have no problem and I'm on Windows too) nor deploy the site correctly. 

What I did instead was update all packages, improve the build process, and retest everything myself. I've now pushed a new update which works nicely locally and [when deployed via Netlify](https://jekyll-tailwindcss-boilerplate.netlify.app/). It *should* work you now I hope.

Could you try fetch the latest changes and retry everything again? Let me know if that works or not.

**Comment by justingolden21** [comment]
Sure, I'll fetch and try again. Thank you so much for not only the quick reply, and the detailed information, but also for updating everything and working with me on this. I'll update you in just a moment.

**Comment by justingolden21** [comment]
As a side note (and I know this possibly should be its own issue but it's an easy fix so I'll just put this here), the website:
https://jekyll-tailwindcss-boilerplate.netlify.app/
for the readme demo link has the tailwind and jekyll links swapped (mixing up clipboard with windows + V maybe?)

Anyway, just thought I'd let you know on that while I'm waiting on install : )

**Comment by justingolden21** [comment]
Hosting succeeded but the styles link is broken (I'm guessing easy fix):

![image](https://user-images.githubusercontent.com/30274440/100423768-64f94f80-3041-11eb-9e98-f3e94dba1caa.png)

Command prompt output in case it's helpfu:

```cmd
Microsoft Windows [Version 10.0.19042.630]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run setup

> your.repo.name@0.0.1 setup C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> gem install bundler && bundle install && npm i

Successfully installed bundler-2.1.4
Parsing documentation for bundler-2.1.4
Done installing documentation for bundler after 3 seconds
1 gem installed
Using public_suffix 4.0.6
Using addressable 2.7.0
Using bundler 2.1.4
Using colorator 1.1.0
Using concurrent-ruby 1.1.7
Using eventmachine 1.2.7
Using http_parser.rb 0.6.0
Using em-websocket 0.5.2
Using ffi 1.13.1 (x64-mingw32)
Using forwardable-extended 2.6.0
Using i18n 1.8.5
Using sassc 2.4.0 (x64-mingw32)
Using jekyll-sass-converter 2.1.0
Using rb-fsevent 0.10.4
Using rb-inotify 0.10.1
Using listen 3.3.1
Using jekyll-watch 2.2.1
Using rexml 3.2.4
Using kramdown 2.3.0
Using kramdown-parser-gfm 1.1.0
Using liquid 4.0.3
Using mercenary 0.4.0
Using pathutil 0.16.2
Using rouge 3.25.0
Using safe_yaml 1.0.5
Using unicode-display_width 1.7.0
Using terminal-table 1.8.0
Using jekyll 4.1.1
Using jekyll-seo-tag 2.7.1
Bundle complete! 2 Gemfile dependencies, 29 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.

> your.repo.name@0.0.1 preinstall C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npx npm-force-resolutions

npx: installed 5 in 2.974s

> deasync@0.1.21 install C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\deasync
> node ./build.js

`win32-x64-node-10` exists; testing
Binary is fine; exiting

> core-js@2.6.12 postinstall C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> parcel-bundler@1.12.4 postinstall C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler
> node -e "console.log('\u001b[35m\u001b[1mLove Parcel? You can now donate to our open collective:\u001b[22m\u001b[39m\n > \u001b[34mhttps://opencollective.com/parcel/donate\u001b[0m')"

Love Parcel? You can now donate to our open collective:
 > https://opencollective.com/parcel/donate
npm WARN notsup Unsupported engine for @tailwindcss/postcss7-compat@2.0.1: wanted: {"node":">=12.13.0"} (current: {"node":"10.16.3","npm":"6.14.9"})
npm WARN notsup Not compatible with your version of node/npm: @tailwindcss/postcss7-compat@2.0.1
npm WARN your.repo.name@0.0.1 No repository field.
npm WARN your.repo.name@0.0.1 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\@parcel\watcher\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

added 766 packages from 429 contributors, removed 66 packages and updated 41 packages in 38.307s

39 packages are looking for funding
  run `npm fund` for details


C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm start

> your.repo.name@0.0.1 start C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> concurrently --kill-others "npm run css:dev" "jekyll serve"

[0]
[0] > your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
[0] > parcel watch index.css --no-source-maps
[0]
Configuration file: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_config.yml
[1]             Source: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate
[1]        Destination: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_site
[1]  Incremental build: disabled. Enable with --incremental
[1]       Generating...
[0] ×  C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\monokai.css:undefined:undefined: getProcessedPlugins is not a function
[0]     at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\tailwindcss\lib\processTailwindFeatures.js:71:83
[0]     at LazyResult.run (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:288:14)
[0]     at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:212:26)
[0]     at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:225:14)
[0]     at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:254:14
[0]     at new Promise (<anonymous>)
[0]     at LazyResult.async (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:250:23)
[0]     at LazyResult.then (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:131:17)
[1]                     done in 5.554 seconds.
[1]   Please add the following to your Gemfile to avoid polling for changes:
[1]     gem 'wdm', '>= 0.1.0' if Gem.win_platform?
[1]  Auto-regeneration: enabled for 'C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate'
[1]     Server address: http://127.0.0.1:4000
[1]   Server running... press ctrl-c to stop.
[1] [2020-11-26 23:44:02] ERROR `/dist/index.css' not found.
[1] [2020-11-26 23:44:11] ERROR `/dist/index.css' not found.

```

**Comment by kangabru** [comment]
You're welcome! The repo was a bit old anyway and needed a refresh.

> for the readme demo link has the tailwind and jekyll links swapped (mixing up clipboard with windows + V maybe?)

Nice find, thanks and fixed.

> Hosting succeeded but the styles link is broken (I'm guessing easy fix):

Ok it seems this happens on very first dev builds. Jekyll and CSS are generated concurrently in dev so on the very first load the CSS file doesn't exist yet and doesn't generate in time. Jekyll therefore doesn't find `dist/index.css`, can't copy the file into the `_site` directory, and thus the styles break.

**To fix this simply open `index.css` while `npm start` is running, save the file, then refresh your browser.** Parcel (which builds the CSS) should automatically update and trigger Jekyll to copy the file.

Once that file exists then future builds should work without issue (and why I didn't catch it while testing). This also doesn't happen in prod because CSS is generated *before* Jekyll so this isn't an issue.

I updated the readme with a warning and solution but if you can think of a better way to solve it I'm all ears.

Hope that helps!

**Comment by justingolden21** [comment]
Thank you again for the reply. When you say to open index.css, do you mean to open in browser or in text editor? Sorry for the dumb question, not sure if you're saying to add to browser cache or for the last saved date on the file to change from text editor... I wasn't able to open it in browser for some reason, and opening in editor and saving doesn't seem to do the trick.

I tried killing and running `npm start` again (I ran `npm run setup` after pulling) and I got errors the second time:

```cmd
Microsoft Windows [Version 10.0.19042.630]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run setup

> your.repo.name@0.0.1 setup C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> gem install bundler && bundle install && npm i

Successfully installed bundler-2.1.4
Parsing documentation for bundler-2.1.4
Done installing documentation for bundler after 7 seconds
1 gem installed
Using public_suffix 4.0.6
Using addressable 2.7.0
Using bundler 2.1.4
Using colorator 1.1.0
Using concurrent-ruby 1.1.7
Using eventmachine 1.2.7
Using http_parser.rb 0.6.0
Using em-websocket 0.5.2
Using ffi 1.13.1 (x64-mingw32)
Using forwardable-extended 2.6.0
Using i18n 1.8.5
Using sassc 2.4.0 (x64-mingw32)
Using jekyll-sass-converter 2.1.0
Using rb-fsevent 0.10.4
Using rb-inotify 0.10.1
Using listen 3.3.1
Using jekyll-watch 2.2.1
Using rexml 3.2.4
Using kramdown 2.3.0
Using kramdown-parser-gfm 1.1.0
Using liquid 4.0.3
Using mercenary 0.4.0
Using pathutil 0.16.2
Using rouge 3.25.0
Using safe_yaml 1.0.5
Using unicode-display_width 1.7.0
Using terminal-table 1.8.0
Using jekyll 4.1.1
Using jekyll-seo-tag 2.7.1
Bundle complete! 2 Gemfile dependencies, 29 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.

> your.repo.name@0.0.1 preinstall C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> npx npm-force-resolutions

npx: installed 5 in 4.305s
npm WARN your.repo.name@0.0.1 No repository field.
npm WARN your.repo.name@0.0.1 No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\@parcel\watcher\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

up to date in 15.66s

41 packages are looking for funding
  run `npm fund` for details


C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm start

> your.repo.name@0.0.1 start C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> concurrently --kill-others "npm run css:dev" "jekyll serve"

[0]
[0] > your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
[0] > parcel watch index.css --no-source-maps
[0]
×  C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\monokai.css:undefined:undefined: getProcessedPlugins is not a function
[0]     at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\tailwindcss\lib\processTailwindFeatures.js:71:83
[0]     at LazyResult.run (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:288:14)
[0]     at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:212:26)
[0]     at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:225:14)
[0]     at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:254:14
[0]     at new Promise (<anonymous>)
[0]     at LazyResult.async (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:250:23)
[0]     at LazyResult.then (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:131:17)
[1] Configuration file: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_config.yml
[1]             Source: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate
[1]        Destination: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_site
[1]  Incremental build: disabled. Enable with --incremental
[1]       Generating...
[1]                     done in 6.032 seconds.
[1]   Please add the following to your Gemfile to avoid polling for changes:
[1]     gem 'wdm', '>= 0.1.0' if Gem.win_platform?
[1]  Auto-regeneration: enabled for 'C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate'
×  The "data" argument must be one of type string, TypedArray, or DataView. Received type object
[0]     at Hash.update (internal/crypto/hash.js:58:11)
[0]     at Bundle.getHash (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\src\Bundle.js:301:12)
[0]     at Bundle.package (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\src\Bundle.js:186:23)
[0]     at Bundler.bundle (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\src\Bundler.js:325:56)
[0]     at process._tickCallback (internal/process/next_tick.js:68:7)
[1] [2020-11-27 23:13:48] ERROR `/dist/index.css' not found.
[1] [2020-11-27 23:13:49] ERROR `/dist/index.css' not found.
[1] [2020-11-27 23:13:49] ERROR `/dist/index.css' not found.
[1] [2020-11-27 23:13:50] ERROR `/dist/index.css' not found.
[1] [2020-11-27 23:14:29] ERROR `/dist/index.css' not found.
[0] Terminate batch job (Y/N)? Terminate batch job (Y/N)? npm run css:dev exited with code 1
--> Sending SIGTERM to other processes..
[1] jekyll serve exited with code 1

^C
C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm start

> your.repo.name@0.0.1 start C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> concurrently --kill-others "npm run css:dev" "jekyll serve"

[0]
[0] > your.repo.name@0.0.1 css:dev C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
[0] > parcel watch index.css --no-source-maps
[0]
Configuration file: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_config.yml
[1]             Source: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate
[1]        Destination: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_site
[1]  Incremental build: disabled. Enable with --incremental
[1]       Generating...
[1]                     done in 0.318 seconds.
[1]   Please add the following to your Gemfile to avoid polling for changes:
[1]     gem 'wdm', '>= 0.1.0' if Gem.win_platform?
[1]  Auto-regeneration: enabled for 'C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate'
[1]     Server address: http://127.0.0.1:4000
[1]   Server running... press ctrl-c to stop.
[0] ×  C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\monokai.css:undefined:undefined: getProcessedPlugins is not a function
[0]     at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\tailwindcss\lib\processTailwindFeatures.js:71:83
[0]     at LazyResult.run (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:288:14)
[0]     at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:212:26)
[0]     at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:225:14)
[0]     at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:254:14
[0]     at new Promise (<anonymous>)
[0]     at LazyResult.async (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:250:23)
[0]     at LazyResult.then (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:131:17)
```

**Comment by justingolden21** [comment]
I tried closing the tab again, killing the command prompt window again, saving changes again and running `npm start` again and the third time (despite errors) it at least said it was serving, the fourth time I did the same steps but this time the errors showed up afterwards, same stuff with LazyResult as above... No idea what it is or why it's happening.

**Comment by justingolden21** [comment]
It seems it's linking to dist/index.css but there's nothing in the dist folder. Opening the index from the _site folder also yields the webpage without styles.

**Comment by kangabru** [comment]
> When you say to open index.css, do you mean to open in browser or in text editor?

Yeah in your local text editor. Just saving that file should cause Parcel to automatically rebuild the file.

Sorry you're having these issues, not sure why it's breaking for you. I tried doing it myself from scratch and everything seems to be working. 

<details>
  <summary>(Click to expand) Here's my console log of me installing it from scratch</summary>

  ```bash
    PS C:\Users\Scott\Documents\Code\boilterplate-jekyll-test> jekyll --version
    jekyll 4.1.1
    PS C:\Users\Scott\Documents\Code\boilterplate-jekyll-test> npm --version
    6.14.8
    PS C:\Users\Scott\Documents\Code> git clone git@github.com:kangabru/jekyll-tailwindcss-boilerplate.git jekyllplate-test
    Cloning into 'jekyllplate-test'...
    remote: Enumerating objects: 128, done.
    remote: Counting objects: 100% (128/128), done.
    remote: Compressing objects: 100% (86/86), done.
    Receiving objects:  68% (88/128)eused 103 (delta 34), pack-reused 0
    Receiving objects: 100% (128/128), 131.61 KiB | 429.00 KiB/s, done.
    Resolving deltas: 100% (55/55), done.
    PS C:\Users\Scott\Documents\Code> cd jekyllplate-test
    PS C:\Users\Scott\Documents\Code\jekyllplate-test> npm run setup

    > your.repo.name@0.0.1 setup C:\Users\Scott\Documents\Code\jekyllplate-test
    > gem install bundler && bundle install && npm i

    Successfully installed bundler-2.1.4
    Parsing documentation for bundler-2.1.4
    Done installing documentation for bundler after 6 seconds
    1 gem installed
    Using public_suffix 4.0.6
    Using addressable 2.7.0
    Using bundler 2.1.4
    Using colorator 1.1.0
    Using concurrent-ruby 1.1.7
    Using eventmachine 1.2.7 (x64-mingw32)
    Using http_parser.rb 0.6.0
    Using em-websocket 0.5.2
    Using ffi 1.13.1 (x64-mingw32)
    Using forwardable-extended 2.6.0
    Using i18n 1.8.5
    Using sassc 2.4.0 (x64-mingw32)
    Using jekyll-sass-converter 2.1.0
    Using rb-fsevent 0.10.4
    Using rb-inotify 0.10.1
    Using listen 3.3.1
    Using jekyll-watch 2.2.1
    Using rexml 3.2.4
    Using kramdown 2.3.0
    Using kramdown-parser-gfm 1.1.0
    Using liquid 4.0.3
    Using mercenary 0.4.0
    Using pathutil 0.16.2
    Using rouge 3.25.0
    Using safe_yaml 1.0.5
    Using unicode-display_width 1.7.0
    Using terminal-table 1.8.0
    Using jekyll 4.1.1
    Using jekyll-seo-tag 2.7.1
    Bundle complete! 2 Gemfile dependencies, 29 gems now installed.
    Use `bundle info [gemname]` to see where a bundled gem is installed.

    > your.repo.name@0.0.1 preinstall C:\Users\Scott\Documents\Code\jekyllplate-test
    > npx npm-force-resolutions

    npx: installed 5 in 4.885s

    > deasync@0.1.21 install C:\Users\Scott\Documents\Code\jekyllplate-test\node_modules\deasync
    > node ./build.js

    `win32-x64-node-14` exists; testing
    Binary is fine; exiting

    > core-js@2.6.12 postinstall C:\Users\Scott\Documents\Code\jekyllplate-test\node_modules\core-js
    > node -e "try{require('./postinstall')}catch(e){}"

    Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

    The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
    > https://opencollective.com/core-js
    > https://www.patreon.com/zloirock

    Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


    > parcel-bundler@1.12.4 postinstall C:\Users\Scott\Documents\Code\jekyllplate-test\node_modules\parcel-bundler
    > node -e "console.log('\u001b[35m\u001b[1mLove Parcel? You can now donate to our open collective:\u001b[22m\u001b[39m\n > \u001b[34mhttps://opencollective.com/parcel/donate\u001b[0m')"

    Love Parcel? You can now donate to our open collective:
    > https://opencollective.com/parcel/donate
    npm WARN your.repo.name@0.0.1 No repository field.
    npm WARN your.repo.name@0.0.1 No license field.
    npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\@parcel\watcher\node_modules\fsevents):
    npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

    added 1010 packages from 504 contributors in 54.115s

    41 packages are looking for funding
    run `npm fund` for details

    PS C:\Users\Scott\Documents\Code\jekyllplate-test> npm start

    > your.repo.name@0.0.1 start C:\Users\Scott\Documents\Code\jekyllplate-test
    > concurrently --kill-others "npm run css:dev" "jekyll serve"

    [0]
    [0] > your.repo.name@0.0.1 css:dev C:\Users\Scott\Documents\Code\jekyllplate-test
    [0] > parcel watch index.css --no-source-maps
    [0]
    Configuration file: C:/Users/Scott/Documents/Code/jekyllplate-test/_config.yml
    [1]             Source: C:/Users/Scott/Documents/Code/jekyllplate-test
    [1]        Destination: C:/Users/Scott/Documents/Code/jekyllplate-test/_site
    [1]  Incremental build: disabled. Enable with --incremental
    [1]       Generating...
    [1]                     done in 5.228 seconds.
    [1]   Please add the following to your Gemfile to avoid polling for changes:
    [1]     gem 'wdm', '>= 0.1.0' if Gem.win_platform?
    [1]  Auto-regeneration: enabled for 'C:/Users/Scott/Documents/Code/jekyllplate-test'
    [1]     Server address: http://127.0.0.1:4000
    [1]   Server running... press ctrl-c to stop.
    [0] √  Built in 15.91s.
  ```
</details>

I just had to do the last step manually (open `index.css` locally and save) and the CSS worked fine.

Deleting the `dist` folder then running `jekyll serve` runs without issue on mine so perhaps you're using an older version.

Here are some things you could try:
- Ensure Jekyll and npm are up to data. My versions are `4.1.1` for Jekyll and `6.14.8` for npm.
- TRY: Run `npm run css:rel` and then `jekyll serve`. That should explicitly generate the CSS file before serving Jekyll. `npm start` *should* work after that.
- OR: Delete the `.cache`, `.jekyll-cache`, `dist`, and `_site` folders and run the commands again
- OR: Start again in a completely new folder. See my console log for the commands I used and responses you should expect.

If none of those work I could even jump on a call quick to help you get up and running. Let me know how you go!

**Comment by justingolden21** [comment]
Deleting dist folder, running jekyll serve, then going to localhost:4000 and updating index.css, the output is fine and the webpage is fine except the styles are still missing:

```cmd
Microsoft Windows [Version 10.0.19042.630]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>jekyll serve
Configuration file: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_config.yml
            Source: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate
       Destination: C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 0.189 seconds.
  Please add the following to your Gemfile to avoid polling for changes:
    gem 'wdm', '>= 0.1.0' if Gem.win_platform?
 Auto-regeneration: enabled for 'C:/Users/justi/OneDrive/Documents/GitHub/kangabru-jekyll-tailwindcss-boilerplate'
    Server address: http://127.0.0.1:4000
  Server running... press ctrl-c to stop.
[2020-11-28 10:50:43] ERROR `/dist/index.css' not found.
[2020-11-28 10:50:58] ERROR `/dist/index.css' not found.

```

One minor version ahead of you on npm:

```
C:\Users\justi>jekyll -v
jekyll 4.1.1

C:\Users\justi>npm -v
6.14.9
```

I tried running `npm run css:rel` and I got errors:

```
Microsoft Windows [Version 10.0.19042.630]
(c) 2020 Microsoft Corporation. All rights reserved.

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run css:rel

> your.repo.name@0.0.1 css:rel C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> parcel build index.css --no-source-maps

×  C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\monokai.css:undefined:undefined: getProcessedPlugins is not a function
    at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\tailwindcss\lib\processTailwindFeatures.js:71:83
    at LazyResult.run (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:288:14)
    at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:212:26)
    at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:225:14)
    at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:254:14
    at new Promise (<anonymous>)
    at LazyResult.async (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:250:23)
    at LazyResult.then (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:131:17)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 css:rel: `parcel build index.css --no-source-maps`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 css:rel script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-28T18_54_52_654Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>npm run css:rel

> your.repo.name@0.0.1 css:rel C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate
> parcel build index.css --no-source-maps

×  C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\monokai.css:undefined:undefined: getProcessedPlugins is not a function
    at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\tailwindcss\lib\processTailwindFeatures.js:71:83
    at LazyResult.run (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:288:14)
    at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:212:26)
    at LazyResult.asyncTick (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:225:14)
    at C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:254:14
    at new Promise (<anonymous>)
    at LazyResult.async (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:250:23)
    at LazyResult.then (C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate\node_modules\parcel-bundler\node_modules\postcss\lib\lazy-result.js:131:17)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your.repo.name@0.0.1 css:rel: `parcel build index.css --no-source-maps`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your.repo.name@0.0.1 css:rel script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\justi\AppData\Roaming\npm-cache\_logs\2020-11-28T18_55_38_209Z-debug.log

C:\Users\justi\OneDrive\Documents\GitHub\kangabru-jekyll-tailwindcss-boilerplate>
```

I tried deleting all folders you listed above (except .cache which didn't exist) and it still got the same errors (see above)

Thank you so much for all the help. I'd be happy to jump on a call sometime. My discord is JonnyG21#2641. I'm in the pacific timezone (CA, US).

**Comment by kangabru** [comment]
We fixed the issue as follows:
- Running Parcel directly revealed the error `Object.entries(…).flatMap is not a function`
- This was happening because Node v10 was installed but flatMap was only introduced in Node v11
- We updated node to v14 and everything worked as expected 🎉
